decisions.go

```diff
@@ -15,19 +15,21 @@ func decisions(nick, msg string) string {
 		return nick + ": Choose what? Usage: !choose choice1 or choice2"
 	}
 
-	return nick + ": The Powers that Be have chosen: " + choice // returning the choice
+	return fmt.Sprintf("%s: The powers that be have chosen: %s", nick, choice) // returning the choice
```

Comment:
```suggestion
	return fmt.Sprintf("%s: The powers that be have chosen %s", nick, choice) // returning the choice
```

I have a weird aversion to using two colons in a sentence. Also, thanks for cleaning up that extra spacing